README.adoc

```diff
@@ -352,22 +352,56 @@ Persistent volume is not formatted when creating the stack, if you have a new
 unformatted volume you can enforce formatting by passing
 `-P prepare_registry=true`.
 
-== Accessing the Web UI
+== Accessing OpenShift
 
-You can get the URL for the OpenShift Console (the web UI) from Heat by running
-`heat output-show my-openshift master_console_url`.
+From user point of view there are two entry points into the deployed OpenShift:
 
-Currently, the UI and the resolution for the public hostnames that will be associated
-to services running in OpenShift is dependent on the DNS created internally by
-these Heat templates.
+* OpenShift console and API URLs: these URLs usually point to the
+  loadbalancer host and can be obtained by:
 
-So to access the UI, you can get the DNS IP address by `heat output-show
-my-openshift dns_ip` and put `nameserver $DNS_IP` as the first entry in your
-`/etc/resolv.conf`.
+```
+heat output-show my-openshift console_url
+heat output-show my-openshift api_url
+```
+
+* Router IP: the IP address which application OpenShift router service
+  listens on. This IP will be used for setting wildcard DNS for
+  .apps.<domain> subdomain. The IP can be obtained by:
+
+```
+heat output-show my-openshift router_ip
+```
+
+== Setting DNS
+
+To make sure that console and API URLs resolving works properly, you have to
```

Comment:
@juddmaltin-dell I added two NOTES bellow which hopefully makes it clear on what DNS server records should be added.